msteams-platform/toolkit/create-resource-bot-microsoft-entra-id.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Create bot resource in Microsoft Entra ID
+author: surbhigupta
+description: Create an Azure bot resource in Microsoft Entra ID to add your bot app to Teams
+ms.topic: how-to
+ms.author: surbhigupta
+ms.localizationpriority: medium
+---
+# Create a bot resource in Microsoft Entra ID
+
+Your Teams bot app can access resources available in Microsoft Entra ID. You need to create a bot resource with Microsoft Entra ID. This process registers your bot app with Azure AI Bot Service and enable the bot app to connect to Teams channels.
+
```

Comment:
Can we add a list of prerequisites the user may need to complete tutorial.

[Example](https://review.learn.microsoft.com/en-us/entra/identity-platform/quickstart-web-app-dotnet-core-sign-in?branch=main).

Reply:
Done.